### PR TITLE
Specified minimum cocoapods version in podspec to 1.10.0

### PR DESCRIPTION
**Pull request by aleksandr-dorofeev**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* ~[ ] Are tests passing locally?~
* [x] Are the files formatted correctly?
*~[ ] Did you add unit tests?~
*~[ ] Did you check UI tests on the sample app? They are not executed on CI.~
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Specified minimum cocoapods version in PLCrashReporter.podspec file.

## Related PRs or issues

[AB88083](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/88083)

### CODE REVIEW

**Review by DmitriyKirakosyan — APPROVED** [review]

**Review by MatkovIvan — DISMISSED** [review]

File: PLCrashReporter.podspec [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Pod::Spec.new do |spec|
+  spec.cocoapods_version = '>= 1.10'
```
Comment (AnastasiaKubova) [inline PLCrashReporter.podspec:2]: Do we need to update the changelog that we raised the minimal version of cocoapods?
Also please update the template in PR description.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## Version 1.10.0
 
+* **[Improvement]** Specified minimum cocoapods version in podspec to 1.10.0.
```
Comment (DmitriyKirakosyan) [inline CHANGELOG.md:5]: This should be under ## Version 1.10.1 (Under development). Version 1.10.0 is already released.

**Comment by DmitriyKirakosyan** [comment]
Please run integration tests on CI for this change.

File: CHANGELOG.md (hunk 2 above)
Comment (aleksandr-dorofeev) [reply CHANGELOG.md]: done

**Review by DmitriyKirakosyan — APPROVED** [review]

**Review by AnastasiaKubova — APPROVED** [review]

**Pull request merged**